### ISSUE TITLE
Sizes of two tensors mismatch when executing {outputs -= means}  in entropy_models.py

### ISSUE THREAD

**Issue by cookie-1921** [InterDigitalInc/CompressAI#116]
When I give the input image with size {224,224,3), this error just happens. I think some efforts are needed to crop one of them. Wish it to be fixed.

**Comment by cookie-1921** [comment]
The mismatch problem also appears at the operation of torch.cat(params, ctx_params) for guassain_params.

**Comment by fracape** [comment]
Hi, please use the template report when submitting this issue:
- steps to reproduce
- environment: python3 -m torch.utils.collect_env
- ...
We don't know what command you used, which model, etc.

**Comment by cookie-1921** [comment]
Hi, thanks for your reply.

- Steps to reproduce.

1. You can use the examples/CompressAI Inference Demo.ipynb and give an input image with size (224, 224). 
2. If the selected pretrained model is mbt2018, and the error will happen when come to `outputs -= means` in compressai/entropy_models/entropy_models.py. 
3. If the selected model is cheng2020_anchor, mismatch error will happen when come to `values = inputs - means` in compressai/entropy_models/entropy_models.py and ` gaussian_params = self.entropy_parameters(torch.cat((params, ctx_params), dim=1) )` in compressai/models/google.py

- My environment

Linux, centos7, pytorch 1.10 for cuda10.2.

**Comment by fracape** [comment]
Hi, 
I see, you image is too small, considering the reduction of dimensionality induced by the autoencoders you used. 
If you use the notebook, it does not contain the padding necessary for very small content. 
Please check example/codec.py or utils.eval_model to see how you can pad the input to still use these models on small content
